package.json

```diff
@@ -10,27 +10,28 @@
     "url": "https://github.com/ampproject/amp-github-apps.git"
   },
   "scripts": {
-    "start": "node build-system/pr-check.js",
-    "lint": "eslint . --ext .ts,.tsx",
-    "fix": "eslint . --ext .ts,.tsx --fix",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "fix": "npm run lint -- --fix",
+    "test": "npm run lint",
     "replace-secrets": "node bin/replace-secrets.js",
     "unredact-env": "read -p \"This will overwrite existing .env file. Press enter to continue.\n\" && node bin/unredact-env.js"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "2.34.0",
-    "@typescript-eslint/parser": "2.34.0",
+    "@typescript-eslint/eslint-plugin": "4.26.0",
```

Comment:
Glad to see you got this to work. I was seeing some strange issues related to a similar version change in [this PR](https://github.com/ampproject/amp-github-apps/pull/1359). Out of curiosity was this related to the `eslint` version?

Reply:
Yes, I'm fairly certain this was due to the variety of plugin versions across different apps, and the fact that they might've been incompatible with the variety of `eslint` / `typescript` / `prettier` versions.